src/ringcentral-web-phone.js

```diff
@@ -157,11 +157,20 @@
         this.appName = options.appName;
         this.appVersion = options.appVersion;
 
+        var outboundProxy = localStorage.getItem('webPhoneOutboundProxy');
```

Comment:
You cannot use local storage from the library. You have to pass it through options.